### PR TITLE
Fix/deleted user jwt

### PR DESCRIPTION
**Pull request by coffemakingtoaster**
Deleted or non active users JWT should no longer be accepted even if they havent expired yet

### CODE REVIEW

**Comment by coffemakingtoaster** [comment]
Vallah wo approval?!?!?!!?!??!?

**Comment by coffemakingtoaster** [comment]
DU SOHN
ISCH WEIß WO DEIN HAUS WOHNT DU OTTO

**Comment by LarsFlieger** [comment]
Fix import and error message

File: src/auth/auth.service.ts [hunk 1]
```diff
@@ -87,4 +89,23 @@ export class AuthService {
             access_token: this.jwtService.sign(payload)
         }
     }
+
+    async isValidJWT(userId: ObjectId): Promise<boolean> {
+        let user: User
```
Comment (LarsFlieger) [inline src/auth/auth.service.ts:94]: We could make a const of that when we move the logic into the try in catch. However it's better readable like that.

Comment (coffemakingtoaster) [reply src/auth/auth.service.ts:94]: No not possible as declaring a const in a try catch block leads to the const not being accessible outside the try catch block

**Pull request merged**